ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -314,7 +314,32 @@ const GENE_DISEASE_DETAIL_SECTIONS = [
   },
 ]
 
-const RNA_SEQ_DETAIL_FIELDS = ['zScore', 'pValue', 'pAdjust']
+const RNA_SEQ_COLUMNS = [
+  { name: 'individual', content: '', width: 3 },
+  { name: 'zScore', content: 'Z-Score', width: 3, format: ({ zScore }) => (zScore ? zScore.toPrecision(3) : null) },
+  { name: 'pValue', content: 'P-Value', width: 3, format: ({ pValue }) => (pValue ? pValue.toPrecision(3) : null) },
+  { name: 'pAdjust', content: 'P-Adjust', width: 3, format: ({ pAdjust }) => (pAdjust ? pAdjust.toPrecision(3) : null) },
+]
+
+const PHENOTYPE_GENE_INFO_COLUMNS = [
+  { name: 'individual', content: '', width: 3 },
+  { name: 'rank', content: 'Rank', width: 3 },
+  { name: 'diseaseName', content: 'Disease', width: 3, format: ({ diseaseName, diseaseId }) => `${diseaseName} (${diseaseId})` },
+]
+
+const PHENOTYPE_GENE_SCORE_COLUMNS = {
+  lirical: [
+    ...PHENOTYPE_GENE_INFO_COLUMNS,
+    { name: 'scores.post_test_probability', content: 'Posttest-Probability', width: 3, format: ({ scores }) => (scores.post_test_probability.toPrecision(3)) },
+    { name: 'scores.compositeLR', content: 'Composite-LR', width: 3, format: ({ scores }) => (scores.compositeLR.toPrecision(3)) },
+  ],
+  exomiser: [
+    ...PHENOTYPE_GENE_INFO_COLUMNS,
+    { name: 'scores.exomiser_score', content: 'Exomiser-Score', width: 3, format: ({ scores }) => (scores.exomiser_score.toPrecision(3)) },
+    { name: 'scores.phenotype_score', content: 'Phenotype-Score', width: 3, format: ({ scores }) => (scores.phenotype_score.toPrecision(3)) },
+    { name: 'scores.variant_score', content: 'Variant-Score', width: 3, format: ({ scores }) => (scores.variant_score.toPrecision(3)) },
+  ],
```

Comment:
Can we use static score fields for LIRICAL and Exomiser tools?

Reply:
no

Reply:
So, can we assume all the score fields are the same for records of a type of tool and generate the columns for the DataTable dynamically?

Reply:
yes